src/main/java/io/xpring/xpring/XpringClient.java

```diff
@@ -0,0 +1,60 @@
+package io.xpring.xpring;
+
+import io.xpring.payid.PayIDClientInterface;
+import io.xpring.payid.PayIDException;
+import io.xpring.xrpl.Wallet;
+import io.xpring.xrpl.XRPClientInterface;
+import io.xpring.xrpl.XRPException;
+
+import java.math.BigInteger;
+
+/**
+ * Composes interactions of Xpring services.
+ */
+public class XpringClient {
+  /**
+   * A {@link PayIDClientInterface} used to interact with the Pay ID protocol.
+   */
+  @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+  private PayIDClientInterface payIDClient;
+
+  /**
+   * A {@link XRPClientInterface} used to interact with the XRP Ledger protocol.
+   */
+  private XRPClientInterface xrpClient;
+
+  /**
+   * Create a new XpringClient.
+   *
+   * @param payIDClient A Pay ID Client used to interact with the Pay ID protocol.
+   * @param xrpClient   An XRP Client used to interact with the XRP Ledger protocol.
+   */
+  @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+  public XpringClient(PayIDClientInterface payIDClient, XRPClientInterface xrpClient) {
+    // TODO(keefertaylor): Verify that given inputs are on the same network.
+    this.payIDClient = payIDClient;
+    this.xrpClient = xrpClient;
+  }
+
+  /**
+   * Transact XRP between two accounts on the ledger.
+   *
+   * @param amount           The number of drops of XRP to send.
+   * @param destinationPayID A destination Pay ID to send the drops to.
+   * @param sourceWallet     The {@link Wallet} which holds the XRP.
+   * @return A transaction hash for the payment.
+   * @throws PayIDException If there was a problem resolving the Pay ID.
+   * @throws XRPException   If there was a problem sending XRP.
+   */
+  public String send(
+      final BigInteger amount,
+      final String destinationPayID,
```

Comment:
I don't know what your decision was around casing for variables and such, so feel free to ignore all these suggestions - I'll just suggest what would be idiomatic in Java.

```suggestion
      final String destinationPayId,
```